variables-extensions.tf

```diff
@@ -1,45 +1,190 @@
 # Copyright (c) 2017, 2023 Oracle Corporation and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
 
-variable "await_node_readiness" {
-  default     = "none"
-  description = "Optionally block completion of Terraform apply until one/all worker nodes become ready."
+# Calico
+
+variable "calico_enabled" {
```

Comment:
please rename to imperative form to remain consistent with other features we are enabling/creating. so it should be `enable_calico`

Reply:
to change in a future PR